install-npm.js

```diff
@@ -32,7 +34,14 @@ function waitForDeps (cb) {
   check();
 }
 
-if (require.main === module) {
+function main () {
+  // check if we should skip install
+  if (!_.isEmpty(process.env.APPIUM_SKIP_CHROMEDRIVER_INSTALL) || !_.isEmpty(process.env.npm_config_chromedriver_skip_install)) {
```

Comment:
I think it also makes sense to add the description of this parameter into the error message, which appears on chromedriver download failure